invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -113,26 +111,27 @@
       margin-top: 7px;
     }
   }
-
 }
 
 .communities-creation {
-  label.helptext{
-    margin-top: -8px;
+  label.helptext {
     margin-bottom: 15px;
     display: block;
     color: #737373;
     font-weight: normal;
     font-size: 0.9em;
   }
 
+  .ui.header:first-child {
```

Comment:
this will affect all the semantic ui headers in the community module or any place which uses its css assets, is this the goal?

Reply:
It was adding negative margin I guess it's needed by the cmp itself so will make it specific :+1: Thanks!